crate_universe/defs.bzl

```diff
@@ -24,7 +17,7 @@ def _input_content_template(ctx, name, packages, cargo_toml_files, overrides, re
         overrides (dict): A dict of crate name (`str`) to json encoded `crate.override` data.
         registry_template (str): A crate registry url template
         targets (list): A list of target platform triples
-        cargo_bin_label (Label): The label of a Cargo binary.
+        cargo_bin_path (Label): The label of a Cargo binary.
```

Comment:
This doc probably needs a little updating - it's now a string (so doesn't need stringifying later?) rather than a Label?

Reply:
Fixed but `cargo_bin_path` is a [path](https://docs.bazel.build/versions/master/skylark/lib/path.html), not a string
```
DEBUG: /private/var/tmp/_bazel_user/5f9b22deab82162fb4591147bb665052/external/rules_rust/crate_universe/defs.bzl:26:10: path
```